life_hc/MySQL/Vehicles/fn_vehicleCreate.sqf

```diff
@@ -7,14 +7,17 @@
     Description:
     Answers the query request to create the vehicle in the database.
 */
-private ["_uid","_side","_type","_classname","_color","_plate"];
-_uid = [_this,0,"",[""]] call BIS_fnc_param;
-_side = [_this,1,sideUnknown,[west]] call BIS_fnc_param;
-_vehicle = [_this,2,objNull,[objNull]] call BIS_fnc_param;
-_color = [_this,3,-1,[0]] call BIS_fnc_param;
+private ["_type","_classname","_plate"];
+params [
+    "_uid",
```

Comment:
it's a string, doesn't need a default value. It'll always be "".
https://community.bistudio.com/wiki/params